src/st/chi/en_gremlin.c

```diff
@@ -13,6 +13,21 @@ static u8 AnimFrames_Running[] = {
     0x06, 0x01, 0x03, 0x02, 0x02, 0x03, 0x02, 0x04, 0x03, 0x05, 0x06, 0x06, 0x03, 0x05, 0x02, 0x04,
     0x02, 0x03, 0x03, 0x02, 0x00, 0x00, 0x00, 0x00
 };
+
+static u8 AnimFrames_Fire[] = {
+    0x01, 0x07, 0x01, 0x08, 0x01, 0x09, 0x01, 0x0A, 0x01, 0x0B, 0x01, 0x0C, 0x01, 0x0D, 0x01, 0x0E,
+    0x01, 0x0F, 0x01, 0x10, 0x00, 0x00, 0x00, 0x00
+};
+
+// D_8018171C
+static u8 AnimFrames_Glow[] = {
+    0x01, 0x11, 0x01, 0x12, 0x00, 0x00, 0x00, 0x00
+};
+
+static u8 AnimFrames_FireIdle[] = {
+    0x02, 0x14, 0x02, 0x15, 0x02, 0x16, 0x02, 0x17, 0x02, 0x18, 0x02, 0x19, 0x02, 0x1A, 0x02, 0x1B,
+    0x02, 0x1C, 0x02, 0x1D, 0x00, 0x00, 0x00, 0x00
+};
```

Comment:
Had to move this data up to before `pad2_anim_debug` so the debug string gets put in the right place.